AirCasting/SessionViews/ShareViewModal.swift

```diff
@@ -0,0 +1,72 @@
+// Created by Lunar on 28/06/2021.
+//
+
+import AirCastingStyling
+import SwiftUI
+
+struct ShareViewModal: View {
+    @Environment(\.presentationMode) var presentationMode
+    @State var email: String = ""
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            VStack(alignment: .leading, spacing: 5) {
+                title
+                description
+            }
+            checkBox
+            shareButton
+            descriptionMail
+            createTextfield(placeholder: "Email", binding: $email)
+                .padding(.vertical)
+            oKButton
+            cancelButton
```

Comment:
Hmm maybe you could add a VStack for ok and cancel buttons and make smaller spacing there, like 5 or 8? Just for those two.